tests/cucumber/steps/steps.js

```diff
@@ -1902,7 +1817,9 @@ module.exports = function getSteps(options) {
 
   When('we make any Pending Transactions By Address call', async function () {
     anyPendingTransactionsByAddressResponse = await doOrDoRaw(
-      this.v2Client.pendingTransactionByAddress()
+      this.v2Client.pendingTransactionByAddress(
+        'MO2H6ZU47Q36GJ6GVHUKGEBEQINN7ZWVACMWZQGIYUOE3RBSRVYHV4ACJI'
```

Comment:
where does it come from?

Reply:
It's an arbitrary address. This specific step has a mock response, so the argument doesn't matter.

Before it was wrong to pass nothing, but it happened to work because we didn't look at the argument too much. Now `.toString()` is called on all arguments to coerce `Address` types to strings, so the undefined argument is a problem now